deploy/charts/trust-manager/Chart.yaml

```diff
@@ -17,4 +17,4 @@ sources:
 - https://github.com/cert-manager/trust-manager
 
 appVersion: v0.4.0
-version: v0.4.0
+version: v0.4.1
```

Comment:
suggestion: I'd probably prefer that we don't change this here, and instead leave this to be done in a separate PR just before a release. Doesn't really matter though because it'll get updated anyway - as long as the tests pass I'm happy to merge